SSA/WellTypedFramework.lean

```diff
@@ -5,8 +5,11 @@ import Mathlib.Data.Option.Basic
 Typeclass for a `baseType` which is a godel code of 
 Lean types.
 -/
-class Godel (BaseType : Type)  : Type 1 where
-  toType : BaseType → Type
+class Godel (β : Type)  : Type 1 where
+  toType : β → Type
+open Godel /- make toType publically visible in module. -/
+
+notation "⟦" x "⟧" => Godel.toType x
```

Comment:
that's a cool notation, why not call the typeclass `DenoteLean` for example, in allusion to this?